### ISSUE TITLE
[Global Styles] Main blue button's colors are not accessible

### ISSUE THREAD

**Issue by davefuentes** [zurb/foundation-sites#7103]
**How can we reproduce this bug?**

1. Go to Global Styles Page
2. Use WebAIM Color Contrast Checker (http://webaim.org/resources/contrastchecker)
3. Enter "FFFFFF" as "foreground" color.
4. Enter "2199e8" as "background" color.

**What did you expect to happen?**
Contrast ratio for normal text should be 4.5:1 or higher
WCAG AA criteria should pass

**What happened instead?**
Contrast ratio is below standard.
See: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=2199e8

**Comment by andycochran** [comment]
:+1:

Do you think many designers will change default colors to those that still meet Level AA? 

I'm sure hardly anyone will meet Level AAA. 

> "…level AAA will have such a high impact on design, that even the W3C claims that most organizations will not be able to achieve that level (as the compromises on design will be too important)" -[stackoverflow](http://stackoverflow.com/questions/2414321/why-wcag-made-3-level-a-aa-and-aaa). 

But Level AA would be great to shoot for with the defaults, even if we expect visual design and brand constraints to limit sites to Level A.

**Comment by andycochran** [comment]
The slightest change that allows AA to pass is `#1a7cbd` (keeping the same blue hue and saturation, changing only the brightness).

**Comment by davefuentes** [comment]
Will create a PR with your suggestion. Let's see if we have quorum

**Comment by andycochran** [comment]
Cool, @davefuentes! 

You might notice I created issues for the other inaccessible button colors. Unfortunately, I think fixing all of these will make vanilla Foundation less aesthetically pleasing. :disappointed:

**Comment by gakimball** [comment]
I'm collecting all of the contrast issues into one: #7387
